fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadTaskInfo.java

```diff
@@ -184,6 +184,8 @@ public long getTimeToExecuteMs() {
 
     public void setMsg(String msg) {
         this.msg = msg;
+        RoutineLoadJob routineLoadJob = routineLoadManager.getJob(jobId);
+        routineLoadJob.setOtherMsg(msg);
```

Comment:
Better to add `txnId`, `label` information to know which task the msg is from?

Reply:
done. 
txn id and task id are added.